dotcom-rendering/src/web/components/BylineLink.tsx

```diff
@@ -56,7 +64,9 @@ const applyCleverOrderingForMatching = (titles: string[]): string[] => {
  * It does this so we can have separate links to both contributors
  */
 export const bylineAsTokens = (byline: string, tags: TagType[]): string[] => {
-	const titles = tags.filter(isContributor).map((c) => c.title);
+	const titles = tags
+		.filter(isContributor)
+		.map((c) => c.title.replaceAll(SPECIAL_REGEX_CHARACTERS, `\\$&`));
```

Comment:
I'm not sure why we're replacing with`\\$&` - does it have a special meaning? Do you think we could add a comment for future travellers like me?

Reply:
`\\` is just an escaped `\`, but you can't just write `\` as that would try to escape the `$`

`$&` is special `replaceAll` syntax which means "the matched string", it lets you include the old string in the new replacement string.

```js
"foo".replaceAll('foo', '$&bar') === 'foobar' // true
```

Will add a comment too!